cinch/roles/jenkins_master/tasks/https.yml

```diff
@@ -18,31 +18,18 @@
       out: jenkins.crt
     - in: "{{ jenkins_ssl_key }}"
       out: jenkins.key
+  when: jenkins_ssl_cert is defined and jenkins_ssl_key is defined
   notify: restart Jenkins
 
 - name: copy ssl certificate to anchors
   copy:
-    src: "{{ jenkins_ssl_cert }}"
+    src: "{{ (jenkins_ssl_cert is defined) | ternary('{{ jenkins_home }}/.ssl/jenkins.crt',
```

Comment:
Now that keytool is out of the picture, is there any need to store certificates in **$JENKINS_HOME/.ssl**?  I always have seen that path used in conjunction with keytool, and to me it seems that we can just always store certificates where nginx expects to read them by default.